src/sql/magic_cmd.py

```diff
@@ -65,8 +78,34 @@ def execute(self, line="", cell="", local_ns=None):
 
             args = parser.parse_args(others)
             return inspect.get_columns(name=args.table, schema=args.schema)
+
+        elif cmd_name == "profile":
+            parser = CmdParser()
+
+            parser.add_argument(
```

Comment:
let's add support for --schema. the command that lists tables and columns supports it, you can use it as a guide